hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -257,55 +329,235 @@
                          gen/once
                          gen/each)})
 
+(defn log-acquire-ok
+  [clientName op]
+  (do
+    (try (Thread/sleep LOGGING_SLEEP_DURATION_MS) (catch Exception _))
```

Comment:
Typically, in Jepsen tests, you want to perform operations as quickly as possible. I'm confused about why you've got all these sleep statements scattered throughout--it might be worth adding some documentation here to explain your approach.